File: NorthwndBackend/NorthwndBackend/Dtos/OrderDto.cs
namespace NorthwndBackend.Dtos;

public class OrderDto
{
    public int OrderId { get; set; }
    public DateTime OrderDate { get; set; }
    public string? Employee { get; set; }
    public DateTime? ShippedDate { get; set; }
    public decimal? Freight { get; set; }
    public string? ShipName { get; set; }
}

File: NorthwndFrontend/src/data/orderDto.ts
export class OrderDto {
    orderId: number;
    orderDate?: Date;
    employee?: string;
    shippedDate?: Date;
    freight?: number;
    shipName?: string;

    constructor(orderId: number, orderDate?: Date, employee?: string, shippedDate?: Date, freight?: number, shipName?: string) {
        this.orderId = orderId;
        this.orderDate = orderDate;
        this.employee = employee;
        this.shippedDate = shippedDate;
        this.freight = freight;
        this.shipName = shipName;
    }
}